cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -149,11 +149,12 @@ function Multiselect( element ) { // eslint-disable-line max-statements
     } );
 
     _searchDom = create( 'input', {
-      className:   BASE_CLASS + '_search ' + TEXT_INPUT_CLASS,
-      type:        'text',
-      placeholder: _placeholder || 'Choose up to five',
-      inside:      _headerDom,
-      id:          _name
+      className:      BASE_CLASS + '_search ' + TEXT_INPUT_CLASS,
+      type:           'text',
+      placeholder:    _placeholder || 'Choose up to five',
+      inside:         _headerDom,
+      id:             _name,
+      'autocomplete': 'off'
```

Comment:
Do you need the quotes around `autocomplete`?

Reply:
I guess not? I removed them in 4f12e8f, and everything still works fine. Not sure why some of the values are quoted and some aren't elsewhere in the file, but I guess it's better to stay consistent within a group of attributes.

Reply:
Yeah the other ones should be removed too. Quotes are only needed if it has a dash or space if I recall. I think the linter probably will note that too. At any rate, for the scope of this PR having them the same within a group is fine.